### ISSUE TITLE
Support for Azure DevOps Server? (on-prem hosted)

### ISSUE THREAD

**Issue by dmitrysamuylovpharo** [microsoft/azure-pipelines-jira#27]
Does this integration support Azure DevOps Server? (on-prem hosted)

Based on all documentation it looks like this integration is specific to Azure hosted managed Azure Pipelines. It would be really great if you could use it with Azure DevOps Server's Pipelines as well which is an on premise hosted version of the same service.

Maybe this is already supported but just not documented?

**Comment by mrweiland** [comment]
I agree would love to have it on-prem JIRA Server that connect to on-prem Azure Devops.

**Comment by Almyfad** [comment]
It would be wonderful to have this for On premise, Any update about this ?

**Comment by Jan-Pleva** [comment]
Hello, I am interested in this feature too. What is the state?
I create issue in Microsoft support, so you can vote to support, here: 
https://developercommunity.visualstudio.com/idea/1190528/jira-integration-for-tfs.html

**Comment by Jan-Pleva** [comment]
+1 for Jira server integration. Also mentioned in issue #17

**Comment by Jan-Pleva** [comment]
@shashban What about this feature? Is there still some blocker from Atlassian, as you mentioned here: ?
https://devblogs.microsoft.com/devops/azure-pipelines-integration-with-jira-software/#comment-535

**Comment by brothers-morrison** [comment]
+1 for this
